README.md

```diff
@@ -364,7 +364,7 @@ const fly = new Fly({
 
 ### Task
 
-A Task receives itself as its first argument. We choose to name the parameter `fly` simply because of convention; of course, you may call it whatever you'd like.
+A Task receives itself as its first argument. We choose to name the parameter `fly` simply because of the convention; of course, you may call it whatever you'd like.
```

Comment:
The use of `the` is wrong here, but you're right -- sounds weird here. 

Let's change to "simply as a convention;".